rpm/Makefile

```diff
@@ -17,13 +17,7 @@ CURL_VERSION=7.52.1
 endif
 endif
 
-ifeq ($(filter sles%,$(DISTRO)),)  # If DISTRO is not sles*.
-vendor:
-	[ -f curl-$(CURL_VERSION).tar.bz2 ] || curl -O https://curl.haxx.se/download/curl-$(CURL_VERSION).tar.bz2
```

Comment:
We also have to fix all parts of the RPM spec that use `bundle_curl` (that's where this downloaded source is used).